app/views/organisations/_form.html.erb

```diff
@@ -1,41 +1,63 @@
-<%= form_for(@organisation) do |f| %>
+<div class="field">
```

Comment:
We'll need to integrate the same error messaging style as the `service` app. But I guess that should be a separate story.